style.css

```diff
@@ -0,0 +1,550 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: inherit;
+}
+
+:root {
+  --darkgreen: #1d313a;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  height: vh;
+}
+
+.project-wrapper {
+  height: 100vh;
+  display: grid;
+  grid-template-columns: minmax(0.5fr, 0.95fr) 2.25fr;
```

Comment:
Would love to know more about how you decided on the fractions to use. Was it just experimentation?